src/org/zaproxy/zap/extension/ascan/ActiveScanAPI.java

```diff
@@ -612,7 +612,11 @@ public ApiResponse handleApiView(String name, JSONObject params)
 					for (Plugin plugin : hp.getCompleted()) {
 						long timeTaken = plugin.getTimeFinished().getTime() - plugin.getTimeStarted().getTime();
 						int reqs = hp.getPluginRequestCount(plugin.getId());
-						hpList.addItem(createPluginProgressEntry(plugin, "Complete", timeTaken, reqs));
+						if (hp.isSkipped(plugin)) {
+							hpList.addItem(createPluginProgressEntry(plugin, hp.getSkippedReason(plugin), timeTaken, reqs));
```

Comment:
- Should check that it really has a reason? (it returns null if not) Otherwise just use the resource message "ascan.progress.label.skipped"?
 - When it has a reason should it be prefixed with the resource message "ascan.progress.label.skippedWithReason"?